### ISSUE TITLE
Add XAF currency

### ISSUE THREAD

**Issue by dev-juju** [Daniel-Ioannou/flutter_currency_picker#38]
XAF is a currency shared by a couple of countries in central Africa (just like EUR is shared by countries in Europe).
My app targets users in those regions.

Could you add support for this currency?

Thanks

**Comment by Daniel-Ioannou** [comment]
Please confirm that the following info is correct. 
```
 {
    "code": "XAF",
    "name": "Central African CFA franc",
    "symbol": "FCFA",
    "flag": "",
    "decimal_digits": 2,
    "number": 950,
    "name_plural": "Central African CFA franc",
    "thousands_separator": ",",
    "decimal_separator": ".",
    "space_between_amount_and_symbol": false,
    "symbol_on_left": true,
  },
```
Also I don't have the flag. For EUR that is also shared by many countries, package use Europe flag (🇪🇺). 
Is there any flag we can use?

**Comment by dev-juju** [comment]
Hi, thanks for looking into this.

The information is correct. 

There's really no flag for the currency as there is no flag for the central African countries. 
Most platforms representing XAF currently either use "XAF" as both the code and flag or use a white flag (e.g 🏳). 
Leaving flag empty is also common...I think it's up to you.

**Comment by Daniel-Ioannou** [comment]
Ok thanks, I will add XAF in the following days.

**Comment by Daniel-Ioannou** [comment]
Released in [v2.0.10](https://pub.dev/packages/currency_picker/versions/2.0.10).
